### PR TITLE
issue #335 - use openjdk in favor of zulu

### PR DESCRIPTION
**Pull request by lmsurpre**
this commit introduces a dependency on https://github.com/marketplace/actions/setup-java-environment-based-on-adoptopenjdk

Signed-off-by: Lee Surprenant <lmsurpre@us.ibm.com>

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -31,125 +31,40 @@ jobs:
...
-    - name: Set up JDK
-      uses: actions/setup-java@v1
+    - name: Set up OpenJDK
+      uses: joschi/setup-jdk@v1
       with:
         java-version: ${{ matrix.java }}
     - name: Build samples
       run: mvn -B install --file fhir-examples/pom.xml
     - name: Build parent without tests
       run: mvn -B install --file fhir-parent/pom.xml -DskipTests
```
Comment (prb112) [inline .github/workflows/build.yml]:
```suggestion
      run: mvn -B install --file fhir-parent/pom.xml -DskipTests --no-transfer-progress
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -31,125 +31,40 @@ jobs:
...
     steps:
     - uses: actions/checkout@v1
-    - name: Set up JDK
-      uses: actions/setup-java@v1
+    - name: Set up OpenJDK
+      uses: joschi/setup-jdk@v1
       with:
         java-version: ${{ matrix.java }}
     - name: Build samples
       run: mvn -B install --file fhir-examples/pom.xml
```
Comment (prb112) [inline .github/workflows/build.yml]:
```suggestion
      run: mvn -B install --file fhir-examples/pom.xml --no-transfer-progress
```

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -31,125 +31,40 @@ jobs:
...
```
Comment (prb112) [inline .github/workflows/build.yml]:
```suggestion
      run: mvn -B install --file fhir-examples/pom.xml --no-transfer-progress
```

File: .github/workflows/build.yml (hunk 1 above)
Comment (lmsurpre) [reply .github/workflows/build.yml]: was the deleted leading space on purpose?  formatting looked right to me

**Pull request merged**